amr-wind/turbulence/LES/OneEqKsgs.H

```diff
@@ -31,6 +31,7 @@ protected:
     //! Turbulence constant
     amrex::Real m_Ce{0.1};
     amrex::Real m_Ceps{0.93};
+
```

Comment:
```suggestion
```

Reply:
Fixed in 8ecbe368e5e6815e45ca2be481488572dfd0ea7f